### PR TITLE
docs/user-guide/config-examples: add spector700's config

### PR DESCRIPTION
**Pull request by spector700**

### CODE REVIEW

File: docs/user-guide/config-examples.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ all of the nixvim options are accessible no matter how you are using it (flake,
 | [pete3n/nixvim-flake] | |
 | [redyf/Neve] | Meticulously crafted custom configuration for Nixvim |
 | [siph/nixvim-flake] | |
+| [spector700/Akari] | Inspired from LazyVim with parts yoinked from other NixVim configs|
```
Comment (GaetanLepage) [inline docs/user-guide/config-examples.md:33]:
```suggestion
| [spector700/Akari] | Inspired from LazyVim with parts yoinked from other NixVim configs |
```

**Review by GaetanLepage — APPROVED** [review]
LGTM thanks !

**Review by MattSturgeon — APPROVED** [review]
Great, thanks!

**Comment by GaetanLepage** [comment]
@mergifyio queue

**Pull request merged**